bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/tools/WaitForTopicValue.java

```diff
@@ -44,20 +45,24 @@
     public WaitForTopicValue(MqttBrokerConnection connection, String topic)
             throws InterruptedException, ExecutionException {
         final MqttMessageSubscriber mqttMessageSubscriber = (t, payload) -> {
-            future.complete(new String(payload));
+            try {
+                future.complete(new String(payload, "UTF-8"));
```

Comment:
Is it possible to use the `String(byte[] bytes, Charset charset)` method with const `StandardCharsets.UTF_8` to omit the `UnsupportedEncodingException`?